src/main/js/models/Row.js

```diff
@@ -72,14 +72,13 @@ const Row = AmpersandModel.extend({
     const self = this;
     return new Promise(
       function (resolve, reject) {
-        const hasCardinality = f.flow(
-          f.get(["column", "constraint", "cardinality"]),
-          f.props(["to", "from"]),
-          f.any((n) => (n || 0) > 0)
+        const isFromCardinalityGreaterThanZero = f.flow(
+          f.getOr(0, ["column", "constraint", "cardinality", "from"]),
+          f.gt(f.__, 0)
```

Comment:
Ist nicht eher die `cardinality.to` interessant?

Reply:
Nein, from müsste meiner Meinung nach schon passen.

Beispiel Kunde (1) <--> (*) Bankkonto.

Wenn ich eine Zeile Bankkonto kopiere, würde ich erwarten, dass mir ein verlinkter Kunde in die neue Zeile eingetragen wird.
Beim Duplizieren eines Kunden, darf schon aus Gründen der Kardinalität, nicht das Konto des einen Kunden in die duplizierte Zeile eingfügt werden.

Ich muss mir also immer from anschauen und wenn from > 0 ist, darf ich die Werte der Spalte nicht mit duplizieren.